### DISCUSSION TITLE
IP addresses

### DISCUSSION THREAD

**Discussion by Mr-Moggle** [GriefPrevention/GriefPrevention#490] [Q&A] [answered]
How do I switch IP address griefing off? I have a class of 15 computers running Minecraft in 1 location all using the same IP address. The grief prevention plugin wont let me open more than 4 computers at a time Is it possible to turn the IP griefing off or raise the limit?

**Comment by RoboMWM** [comment]
Is this resolved or did you accidentally close this??

**Comment by Mr-Moggle** [comment]
Hi, It is closed, i spent ages trying to find out how do remove the ip limit before creating a ticket. and as soon as i created the ticket i found the location of the setting that i needed to change. 

But thanks for checking. :)

**Comment by mitchngwen** [comment]
I am having a the same issue except I can not for the life of me figure out where in the config it is.  So where and how can I change this?

**Comment by RoboMWM** [comment, marked as answer]
Starts off as Maxipaddresses
